tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html

```diff
@@ -3176,8 +3176,8 @@ <h2>Show similarity to selected datapoint</h2>
                 const modelInferenceValueStr =
                   this.strWithModelName_(inferenceValueStr, modelInd);
                 let count = statsActual[item[modelInferenceValueStr]];
-                allLabels.add(item[modelInferenceValueStr]);
-                allLabels.add(item[modelInferenceValueStr]);
+                allLabels.add(String(item[modelInferenceValueStr]));
```

Comment:
Curious: what happens if you don't explicitly cast it to string?

Reply:
For users that provide "ground truth class ID" as a number to WIT, you may get a confusion matrix where rows and columns are doubled, that is that there is a row for Predicted class 0 (as an int) and a row for Prediction class 0 (as a string) separately. They both contain the same correct numbers for those confusion matrix cells, but the rows and columns are duplicated leading to an overly large and confusing confusing matrix.